build.sbt

```diff
@@ -274,6 +274,11 @@ lazy val oauth = smallModule("oauth",
   Seq(autoconfig) ++ reactivemongo.bundle ++ macwire.bundle
 )
 
+lazy val oauth2 = smallModule("oauth2",
+  Seq(common),
+  Seq()
+)
```

Comment:
do we need a new module for this?

Reply:
probably not, eventually. still too early to decide/review